SocketRocket/SRWebSocket.m

```diff
@@ -563,14 +583,27 @@ - (void)_failWithError:(NSError *)error;
     });
 }
 
-- (void)_writeData:(NSData *)data;
+- (void)_writeData:(NSData *)data
 {
-    [self assertOnWorkQueue];
+    [self _writeData:data complete:NULL];
+}
 
+- (void)_writeData:(NSData *)data complete:(SRSendCompleteBlock)complete {
```

Comment:
And another nit on top of this:
The opening bracket should be on the next line, per common style of the surrounding code.